src/main/java/com/esri/samples/map/create_and_save_map/README.md

```diff
@@ -20,19 +20,18 @@ chosen folder.</p>
   <li>Create an <code>ArcGISMap</code> with a <code>Basemap</code> and operational layers</li>
   <li>Create an <code>OAuthConfiguration</code> with your portal and app credential.</li>
   <li>Add the configuration to the <code>AuthenticationMangager</code></li>
-  <li>Create a <code>Portal</code> and load it. Use a custom <code>AuthenticationChallengeHandler</code> to 
-  authenticate with your username and password</li>
+  <li>Create a <code>Portal</code> and load it. Use a <code>DefaultAuthenticationChallengeHandler</code> to authenticate with your username and password</li>
   <li>Once authenticated, save the map by calling <code>map.saveMapAsAsync()</code>, passing in the title, tags, 
   description, and portal folder</li>
 </ol>
 
 <h2>Relevant API</h2>
 
 <ul>
-  <li>AuthenticationChallengeHandler</li>
   <li>ArcGISMap</li>
   <li>ArcGISMapImageLayer</li>
   <li>Basemap</li>
+  <li>DefaultAuthenticationChallengeHandler</li>
```

Comment:
I don't think we need the included Additional Information section anymore, now that we are using the `DefaultAuthenticationChallengeHandler`? The information about using &display=classic is now redundant.

Reply:
You are 100% right, I could have sworn I deleted this.
Also added `AuthenticationManager` to the Readme as it's quite an important bit of the API here